pgml-dashboard/app/templates/base.html

```diff
@@ -21,6 +21,12 @@
         <script defer type="text/javascript" src="{% static '/js/vendor/lodash-4.17.21.min.js' %}"></script>
         <script defer type="module" src="https://unpkg.com/stimulus@2.0.0/dist/stimulus.umd.js"></script>
         <script defer type="module" src="https://cdn.skypack.dev/pin/@hotwired/turbo@v7.0.0-rc.1-Vumm1JJjKMC0HM7D9Tjd/mode=imports/optimized/@hotwired/turbo.js"></script>
+        <script defer type="module" src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/6.65.7/codemirror.min.js"></script>
```

Comment:
How are these generated with the integrity values?

Reply:
Just a link from that site that indexes all the CDNs for js/css libs.